components/src/core/drawer/drawer-body/drawer-body.component.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { DrawerBodyComponent } from './drawer-body.component';
+import { DrawerBodyModule } from './drawer-body.module';
+import { Component } from '@angular/core';
+
+@Component({
+    template: `
+        <pxb-drawer-body> </pxb-drawer-body>
+    `,
+})
+class TestDrawerBody {}
+
+describe('DrawerBodyComponent', () => {
+    let component: DrawerBodyComponent;
+    let fixture: ComponentFixture<DrawerBodyComponent>;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [TestDrawerBody],
```

Comment:
For this particular example, TestDrawerBody is not needed since there's no content being projected into it for the `it creates` test.